src/server/scripts/Commands/cs_guild.cpp

```diff
@@ -206,6 +207,55 @@ class guild_commandscript : public CommandScript
         return targetGuild->ChangeMemberRank(player->GetGUID(), rank);
     }
 
+    static bool HandleGuildRenameCommand(ChatHandler* handler, char const* _args)
```

Comment:
no no no, please use new cc api

Reply:
I will not convert the whole file for this.

Reply:
It's fine